### DISCUSSION TITLE
Laragon won't execute on my Windows 10

### DISCUSSION THREAD

**Discussion by maridanne** [leokhoa/laragon#456] [Q&A]
I installed Laragon but I can't execute it though I already tried running it as an administrator. I also disabled my antivirus. I uninstalled xampp but I have Django installed for some of my created projects. 
How to troubleshoot this problem? Thank you.

**Comment by Prymeklickng** [comment]
Hi Maridanne,
Are you still experiencing the issue with Laragon?

**Reply by maridanne** [reply]
Hello Prymeklickng,
Thanks for responding, I uninstalled the Laragon-Full and installed the Laragon-Portable instead. It executes until I click any button in the program.

**Comment by tienloc1** [comment]
The laragon works fine for me even if I change the OS hardware (I don't install it on C disk), not sure what's going on for you. No error shows?
